javaslang/src/test/java/javaslang/collection/IntMultimap.java

```diff
@@ -313,8 +313,14 @@ public int characteristics() {
 
     @Override
     public <U> Seq<Tuple2<T, U>> zip(Iterable<? extends U> that) {
+        return zipWith(that, Tuple::of);
+    }
+
+    @Override
+    public <U, R> Seq<R> zipWith(Iterable<? extends U> that, BiFunction<? super T, ? super U, ? extends R> mapper) {
         Objects.requireNonNull(that, "that is null");
```

Comment:
`that is null` doesn't sound very useful to me.
Could we rename `that` to `target` or something less context dependent :)?